substantial/workflows/context.py

```diff
@@ -144,3 +149,23 @@ def cancel_run(self):
         Interrupt after the call
         """
         raise CancelWorkflow(self.run)
+
+
+class Utils:
+    def __init__(self, ctx: Context):
+        self.__ctx = ctx
+
+    async def now(self, tz: Optional[timezone] = None) -> datetime:
+        now = await self.__ctx.save(lambda: datetime.now(tz))
+        return now
+
+    async def random(self, a: int, b: int) -> int:
+        return await self.__ctx.save(lambda: random.randint(a, b))
+
+    async def uuid4(self) -> uuid.UUID:
+        serialized_uuid = await self.__ctx.save(lambda: uuid.uuid4())
+        return serialized_uuid
+
+    @staticmethod
+    def log(level, msg, *args, **kwargs) -> None:
+        logging.log(level, msg, *args, **kwargs)
```

Comment:
Are you planning on making this "durable"?

Reply:
no, i think it's ur pr and may should remove it